pkg/resource/deploy/step_generator.go

```diff
@@ -555,7 +555,96 @@ func (sg *stepGenerator) GenerateDeletes() []Step {
 			}
 		}
 	}
-	return dels
+
+	if len(targets) > 0 {
+		logging.V(7).Infof("Planner was asked to only delete '%v'", targets)
+
+		resourcesToDelete, res := sg.computeResourcesToDelete(targets)
+		if res != nil {
+			return nil, res
+		}
+
+		keys := []resource.URN{}
```

Comment:
If we're doing this strictly for logging, can we throw it under a verbosity check?

Reply:
sure.